sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/DDLParserSuite.scala

```diff
@@ -34,6 +34,10 @@ import org.apache.spark.unsafe.types.{CalendarInterval, UTF8String}
 class DDLParserSuite extends AnalysisTest {
   import CatalystSqlParser._
 
+  private def parseException(sqlText: String): SparkThrowable = {
+    intercept[ParseException](parsePlan(sqlText))
+  }
```

Comment:
Is this function can be shared with other Suites (e.g. https://github.com/apache/spark/pull/38265/) so move this one to `AnalysisTest`?

Reply:
Yes, i will move it

Reply:
Done